flex_common.py

```diff
@@ -0,0 +1,59 @@
+from collections import namedtuple
+from conans import AutoToolsBuildEnvironment, tools
+from conans.util.env_reader import get_env
+import os
+import shutil
+import tempfile
+
+Options = namedtuple("Options", ("shared", "fPIC", ))
```

Comment:
I think the _class_ `Options` is used nowhere

Reply:
Indeed! Nowhere used!